core-tests/src/test/java/org/mule/runtime/core/el/DefaultBindingContextBuilderTestCase.java

```diff
@@ -39,14 +42,24 @@ public void addsBinding() {
 
   @Test
   public void addsBindings() {
```

Comment:
I think we need a specific tests for this, specially one for the case where the context is built from another, to make sure modules are copied.